### PR TITLE
Some writecache fixes

### PR DESCRIPTION
**Pull request by fyrchik**
1. Allow to fetch data on demand in `FSTree`.
2. Initialize flush mark LRU cache on startup.

### CODE REVIEW

File: pkg/local_object_storage/writecache/flush.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ const (
 	// and cache persist.
 	flushBatchSize = 512
 	// flushWorkersCount is number of workers for putting objects in main storage.
```
Comment (carpawell) [inline pkg/local_object_storage/writecache/flush.go:23]: lets rename var name in the comment too

File: pkg/local_object_storage/blobstor/fstree/fstree.go [hunk 2]
```diff
@@ -84,6 +85,13 @@ func (p *IterationPrm) WithHandler(f func(addr *addressSDK.Address, data []byte)
 	return p
 }
 
+// WithLazyHandler sets a function to call on each object.
+// Second function will return file called if called. Otherwise, file is not read at all.
```
Comment (carpawell) [inline pkg/local_object_storage/blobstor/fstree/fstree.go:89]: does that line contain correct statements? if so, could you, please, clarify the meaning, i did not get it

Comment (fyrchik) [reply pkg/local_object_storage/blobstor/fstree/fstree.go:89]: No, forgot to change

File: pkg/local_object_storage/writecache/flush.go (hunk 1 above)
Comment (fyrchik) [reply pkg/local_object_storage/writecache/flush.go:23]: Fixed

File: pkg/local_object_storage/blobstor/fstree/fstree.go (hunk 2 above)
Comment (fyrchik) [reply pkg/local_object_storage/blobstor/fstree/fstree.go:89]: Fixed

**Review by carpawell — DISMISSED** [review]

**Review by cthulhu-rider — APPROVED** [review]

**Review by carpawell — APPROVED** [review]

**Pull request merged**